Frontend/src/index.jsx

```diff
@@ -30,8 +31,10 @@ const router = createBrowserRouter([
         element: (
           <Container>
             <Competition>
-              <PageTabs />
-              <Outlet />
+              <User>
+                <PageTabs />
+                <Outlet />
+              </User>
             </Competition>
```

Comment:
`User` should be something `UserProvider` or something, to make it clear it's just providing the context/data, and not actually some user-based UI which is displaying the children somewhere within it. Probably same with `Competition`.

How come the user provider is inside the competition provider? I would think the user provider would be pretty close to the root, since it sounds like the kind of thing that doesn't change much (only on login/logout), whereas once logged in a user would navigate between multiple competitions.

Reply:
It should be closer to the root, but the permissions context depends on competitions, so it could be Users -> Competitions -> Permissions?

Reply:
Currently a user needs to go back to the monolith to change comps so we sadly don't get any benefits there. I will make the users/me route return a 304 though, so at least in most cases it doesn't need to refetch

Reply:
Oh, I'm not really familiar with the integration into the monolith.
So is the route `/competitions` handled by the monolith or by wca-registration? Looking at the code for the router I thought it was handled here, but visiting the test site it seems be the monolith.

Reply:
yeah /competitions is currently handled by the monolith. We do want to move that page to react eventually as part of the competitions service and it would be nice to have some data sharing there, but we see when we get there

Reply:
I get the confusing though looking at the router, notice that there is no /competition -> children -> / route!